src/commands/data/privatelink/access/index.ts

```diff
@@ -1,42 +1,43 @@
 import {flags} from '@heroku-cli/command'
-import {cli} from 'cli-ux'
-
+import {Args, ux} from '@oclif/core'
 import BaseCommand, {PrivateLinkDB} from '../../../../base'
 import fetcher from '../../../../lib/fetcher'
 
-export default class EndpointsAccessIndex extends BaseCommand {
+export default class Index extends BaseCommand {
+  static topic = 'data:privatelink:access'
   static description = 'list all allowed accounts for your privatelink endpoint'
-  static aliases = ['pg:privatelink:access', 'kafka:privatelink:access', 'redis:privatelink:access']
+  static hiddenAliases = ['pg:privatelink:access', 'kafka:privatelink:access', 'redis:privatelink:access']
```

Comment:
I noticed we switched these to `hiddenAliases`.. is there a reason we did that?

Reply:
Truly, I don't remember exactly what my train of thoughts was at the moment I did the change. I guess that my idea was to not introduce a breaking change, but also don't publish the alias under the `pg` (core topic), `kafka`  and `redis` topics because those topics aren't being defined by this plugin. But I don't think is a big deal. We can revert the change and leave it as is.

Reply:
If we make them not hidden, those aliases will show up as commands in the README. We may not want that.

Reply:
Ah ok. There will be change here, though. Currently, those aliases show up when you do `heroku help data:privatelink:access:add` for instance. When we declare them as `hiddenAliases` they won't show up in the help. I don't have a strong opinion here, but maybe worth asking data or someone from product.

Reply:
I can ask about this, good call.

Reply:
after convo with data it sounds like at least keeping them hidden is actually preferable.